test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -52,6 +53,18 @@ get_stat(scoped_cursor &cursor, int stat_field, int64_t *valuep)
     testutil_check(cursor->reset(cursor.get()));
 }
 
+static std::string
+collection_name_to_file_name(const std::string &collection_name)
```

Comment:
So statics don't go in the implementation files?

Reply:
Moved to .cxx file as they're only used by this class.